test/jdk/java/awt/List/ListEnterExitTest.java

```diff
@@ -66,19 +70,23 @@ public void start() throws Exception {
             });
 
             final Robot robot = new Robot();
-            robot.delay(1000);
             robot.waitForIdle();
+            robot.delay(1000);
 
             EventQueue.invokeAndWait(() -> {
                 p = list.getLocationOnScreen();
             });
-            robot.mouseMove(p.x + 10, p.y + 10);
+            robot.mouseMove(p.x + 30, p.y + 40);
+            robot.waitForIdle();
             robot.delay(100);
+
+            robot.mouseMove(p.x - 30, p.y + 40);
             robot.waitForIdle();
-            robot.mouseMove(p.x - 10, p.y - 10);
             robot.delay(100);
+
+            robot.mouseMove(p.x + 30, p.y + 40);
             robot.waitForIdle();
-            robot.mouseMove(p.x + 10, p.y + 10);
+            robot.delay(100);
 
```

Comment:
I don't really understand why `waitForIdle` and `delay` are needed here at all.

Robot just moves mouse to generate mouse enter and exit events. It can do so without any delays, can't it?

Reply:
There is an intermittent failure observed only on macOS 14.7.1 machines. So, I added a small delay and after that on multiple runs in CI, I don't see any failures.